app/controllers/api/v2/allocations_actions.rb

```diff
@@ -0,0 +1,7 @@
+module Api::V2
+  module AllocationsActions
+    def included_relationships
+      IncludeParamHandler.new(params).call
+    end
+  end
+end
```

Comment:
I don't think you need this file - for V2 presumably the plan is to not return any included relationships by default, which is covered by the same method in `ApiController`. This module doesn't change that method implementation at all so looks redundant to me. For V1 the equivalent module is overriding the implementation from `ApiController` so that makes sense.